org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xsd/participants/XSDErrorCode.java

```diff
@@ -34,6 +34,9 @@
  *
  * @see https://wiki.xmldation.com/Support/Validator
  *
+ * All error code types and messages can be found in the Xerces library
+ * http://svn.apache.org/viewvc/xerces/java/trunk/src/org/apache/xerces/impl/msg/XMLSchemaMessages.properties?revision=1869198&view

```

Comment:
I think it's better to use github link https://github.com/apache/xerces2-j/blob/trunk/src/org/apache/xerces/impl/msg/XMLSchemaMessages.properties because you can nanvigate easily for another sources.

Reply:
Agreed, though I noticed the GitHub link doesn't seem as up to date as the SVN one. I'd double check that just to be sure updates still flow through there.